app/views/search/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<br>
+<br>
+Searched for '<%= @query.titleize %>' Quotes
+<br>
+<br>
+Top Results: <br>
+<hr>
+<% @top_results.each do |quote| %>
+  <i>"<%= quote.content %>" </i> <br>
+  - <b> <%= quote.author %>, </b> 
+  tags: <% quote.tags.each do |tag| %>
+          <%= tag.titleize  %>
+        <% end %> <br>
+         <br>
+<% end %>
+<hr>
+
+Total Results: <%= number_with_delimiter(@result_count) %><br>
```

Comment:
what is `number_with_delimiter` doing? When I spin up localhost, your total results are: 2,042. That seems like a lot. When I use Postman and search with the word `love` I am only getting a `:totalCount` of 122.

Reply:
It adds a comma if it is a 4 digit number